src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,646 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Markdown from 'markdown-to-jsx';
+import Draggable from 'react-draggable';
+
+import { ajax } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+
+export class MdSortableTable extends React.PureComponent {
+
+    static propTypes = {
+        'mdFilePath' : PropTypes.string,
+        'content': PropTypes.string,
```

Comment:
(Low importance)

_Theoretically_ we may be able to use "children" prop instead of "content" which would allow us to define the content more like:

```
<MdSortableTable>
| Abcd | efgh | etc. etc. |
+------+------+-----------+
</MdSortableTable>

-or- (maybe if above doesn't work / become accessible as string..)

<MdSortableTable>{'
| Abcd | efgh | etc. etc. |
+------+------+-----------+
'}</MdSortableTable>
```
And then use props.children instead of props.content in constructor etc, with checks to ensure that is of type string.
(Up to you if want to try/adjust)

Reply:
@alexkb0009 At first, have tried the both solutions for inline content, but not succeedded. Then gave up and use `content` prop instead of children. Your comment makes me try it once more, but this time I set `JsxParser`'s `showWarnings` (https://www.npmjs.com/package/react-jsx-parser#proptypes--settings) prop as `True` and get "unterminated string literal exception".

Add escape end of each line and (also do not forget to escape single quotation if exists content) as below and this time it works:

```
<MdSortableTable>{' \
| Abcd | efgh | etc. etc. | \
+------+------+-----------+ \
'}</MdSortableTable>
```

But markdown table editor may forget to add escapes when creating `JSX`, and it is not trivial to find what prevents rendering. Instead I tried to use `string literals/templates` but it is not working, idk why `props.children` is `undefined`.

```
<MdSortableTable>{`
| Abcd | efgh | etc. etc. |
+------+------+-----------+
`}</MdSortableTable>
```

`string literals/templates` approach would make MD editor's changes fast, otherwise using `mdFilePath` is safer.